status/status.go

```diff
@@ -78,7 +78,8 @@ func FromProto(s *spb.Status) *Status {
 //
 //   - If err was produced by this package or implements the method `GRPCStatus()
 //     *Status`, or if err wraps a type satisfying this, the appropriate Status is
-//     returned.
+//     returned.  For wrapped errors, the message returned contains the entire
+//     err.Error() text and not just the wrapped status.
 //
 //   - If err is nil, a Status is returned with codes.OK and no message.
```

Comment:
Nit: can we explicitly specify it returns true in these two cases, and not use the fact that this is the if on the else of the third case which returns false

Reply:
IMO this is such a common pattern that the extra sentences to document "ok" wouldn't be worth worth the clutter they would create.  E.g. this is the documentation of context.Deadline():
```
	// Deadline returns the time when work done on behalf of this context
	// should be canceled. Deadline returns ok==false when no deadline is
	// set. Successive calls to Deadline return the same results.
```

Reply:
Ok fair enough